ghettoVCB.sh

```diff
@@ -903,7 +903,7 @@ ghettoVCB() {
             #1 = readonly
             #0 = readwrite
             logger "debug" "Mounting NFS: ${NFS_SERVER}:${NFS_MOUNT} to /vmfs/volume/${NFS_LOCAL_NAME}"
-	    if [[ ${ESX_RELEASE} == "5.5.0" ]] || [[ ${ESX_RELEASE} == "6.0.0" || ${ESX_RELEASE} == "6.5.0" || ${ESX_RELEASE} == "6.7.0" || ${ESX_RELEASE} == "7.0.0" || ${ESX_RELEASE} == "7.0.1" || ${ESX_RELEASE} == "7.0.2" ]] ; then
+	    if [[ ${ESX_RELEASE} == "5.5.0" || ${ESX_RELEASE} == "6.0.0" || ${ESX_RELEASE} == "6.5.0" || ${ESX_RELEASE} == "6.7.0" || ${ESX_RELEASE} =~ "^7\.0\." ]] ; then
```

Comment:
The VER variable appears nowhere in the script - how to you get it? ESX_RELEASE is set by an explicit call of `uname -r` - so that's what I worked with.

Reply:
Oh I see, VER only seems to be available in the context of [`sanityCheck()`](https://github.com/lamw/ghettoVCB/blob/58e75769f556cd489b9b3b1a98b23117445fcfd5/ghettoVCB.sh#L310)

Reply:
Any chance to see this merged?